### ISSUE TITLE
Does not seem to work in Microsoft Edge (Build 20.10240.16384.0)

### ISSUE THREAD

**Issue by webdesignberlin** [Nickersoft/push.js#9]
[Demo](http://nickersoft.github.io/push.js/demo.html) doesn´t work on Windows 10 with Edge Browser.

```
PushError: push.js is incompatible with self browser.
push.js (357,17)
```

**Comment by matjaz** [comment]
almost! :)

http://caniuse.com/#search=notifications
https://developer.microsoft.com/en-us/microsoft-edge/platform/status/webnotifications

**Comment by RinkAttendant6** [comment]
IMO it's a little misleading to claim support for IE 9+ when the feature only works on EdgeHTML 14+

**Comment by matjaz** [comment]
ie uses mssitemode which is not standard but is ie specific. It was removed from edge as a lot of other ie specific features. ie and edge are different browsers.

**Comment by amnah** [comment]
It doesn't seem to work in IE at all.

Using IE 11.0.10240.16384, and testing with IE9 and IE10 in emulation. None of them work - no errors and no console messages.

I also tried to pin the site by dragging the icon (left of url) to taskbar, but still didn't get any desktop notifications.

**Comment by Nickersoft** [comment]
@amnah I see that now. According to the [caniuse website](http://caniuse.com/#search=notifications) there does not seem to be any support at all for IE. I can look into the issue (maybe there's just a bug with how msSiteMode is being called), but if I can't find anything I'll just remove IE from the list of supported browsers, as it was never officially supported in the browser to begin with.

**Comment by Nickersoft** [comment]
So as a PR by @hbarcelos brought to my attention, the code involved in managing notifications using msSiteMode (as well as webkit for that matter) referenced a variable which was removed versions ago. This issue was fixed in the latest version (v0.0.9), so I would try using the library in IE/Edge again and see if it works. If not, I'm afraid it's probably a lost cause at this point.

**Comment by theLufenk** [comment]
IE has no support for Notifications. Edge is supported by Push.js.
